Offseason2022/src/main/java/frc/robot/Constants.java

```diff
@@ -244,15 +244,20 @@
   public static final class VIConsts
   {
     // Limelight-defined streaming states
-    public static final int    STANDARD       = 0;  // Both cameras side-by-side
-    public static final int    PIP_MAIN       = 1;  // Limelight with second camera inset
-    public static final int    PIP_SECONDARY  = 2;  // Second camera with limelight inset
+    public static final int STANDARD        = 0;  // Both cameras side-by-side
+    public static final int PIP_MAIN        = 1;  // Limelight with second camera inset
+    public static final int PIP_SECONDARY   = 2;  // Second camera with limelight inset
 
     // Limelight-defined LED mode states
-    public static final int    LED_CUR_MODE   = 0;
-    public static final int    LED_OFF        = 1;
-    public static final int    LED_BLINK      = 2;
-    public static final int    LED_ON         = 3;
+    public static final int LL_LED_CUR_MODE = 0;
+    public static final int LL_LED_OFF      = 1;
+    public static final int LL_LED_BLINK    = 2;
+    public static final int LL_LED_ON       = 3;
+
+    public enum VI_LLRequests
+    {
+      VISION_OFF, VISION_ON, VISION_BLINK, VISION_CUR_MODE, VISION_TOGGLE
```

Comment:
Since we're never "blinking" the vision system or requesting the "CUR_MODE", just delete those. And change VI_LLRequests to just VIRequest. But the rest of the vision stuff is good and done.